test/sizes.gzip.json

```diff
@@ -0,0 +1,43 @@
+{
+  "amp.js": "92.33 kB",
```

Comment:
Yeah. Sorry for the switcheroo, but it does sound better at this point. We really don't need to have both.